services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hanke.kt

```diff
@@ -92,6 +88,21 @@ data class Hanke(
             null
         }
 
+    // -------------- Tormaystarkastelu -------------
+    /**
+     * Liikennehaittaindeksi is always available here if it has been calculated and is still valid.
+     * Might be around even if no longer valid.
+     * TODO: should the validity-state be part of this indeksitype, too? (I.e. would know that even
+     *  when not getting the full tulos-data (which does have that state).
+     */
+    var liikennehaittaindeksi: LiikennehaittaIndeksiType? = null
```

Comment:
Maybe  change also here Type = Tyyppi   ?

Reply:
Since that is class name, it doesn't show up anywhere else, and I thought it was used as "this is a type/class"-part of the class name (since the content of that class is not only about the type of chosen index), in which case it would be ok to stay in English.  I'd rather just drop the Type-word out completely, since we don't use it in that purpose elsewhere in the code.